internal/uvm/create_lcow.go

```diff
@@ -115,6 +118,7 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		OutputHandler:         parseLogrus(id),
 		VPMemDeviceCount:      DefaultVPMEMCount,
 		VPMemSizeBytes:        DefaultVPMemSizeBytes,
+		VPMemOptimized:        false,
```

Comment:
What OS build is the pmem packing supported on?

Reply:
I'd imagine we'd want this on by default and with an annotation to go back to the old behavior but I'm probably missing something important

Reply:
I think minimal version is 19H1